src/app/pages/home/home-routing.module.ts

```diff
@@ -11,44 +11,25 @@ const routes: Routes = [
     component: HomePage,
     children: [
       {
-        path:'rolodex',
-        children: [
-          {
-            path:'',
-            loadChildren:'./rolodex/rolodex.module#RolodexPageModule'
-          }
-        ]
+        path:'rolodex', loadChildren:'./rolodex/rolodex.module#RolodexPageModule'
```

Comment:
I'm not sure the rules for nested routing modules but I don't think there need to be children lists at this level. More research here is needed.